hydro_deploy/core/src/kubernetes/launched_binary.rs

```diff
@@ -0,0 +1,102 @@
+#[cfg(unix)]
+use std::sync::Arc;
+
+use async_channel::{Receiver, Sender};
```

Comment:
I think @MingweiSamuel replaced this with a tokio builtin if I remember correctly?? Will need to check against the SSH implementation.